scripts/docker-compose/imports/smartdocuments-wiremock/mappings/wsxmldeposit-wizard.json

```diff
@@ -53,7 +53,7 @@
             "omschrijving": "dummyZaakOmschrijving",
             "registratiedatum": "${json-unit.any-string}",
             "startdatum": "01-01-1970",
-            "status": "Intake",
+            "status": "Aanvullende informatie vereist",
```

Comment:
Do we replace the `Intake` status, or is that not used?

Reply:
There's no call with `Intake`. Instead now we have a call for `Aanvullende informatie vereist` as the zaak under test has additional info task